src/ServiceInsight/MessageList/MessageListViewModel.cs

```diff
@@ -76,7 +75,9 @@ public class MessageListViewModel : RxConductor<RxScreen>.Collection.AllActive,
 
         public MessageSelectionContext Selection { get; }
 
-        public bool WorkInProgress => workCount > 0 && !Parent.AutoRefresh;
+        public int WorkCount { get; private set; }
```

Comment:
The previous implementation had this `WorkCount` as internal. While it is made public is it used anywhere? AFAIK all we bind to is `WorklInProgress` propery.